source/_docs/terminus/install.md

```diff
@@ -0,0 +1,84 @@
+      <figure><pre id="mac-install-composer"><code class="bash" data-lang="bash">cd $HOME ; mkdir terminus1x ; cd terminus1x ; echo '{"minimum-stability": "dev", "prefer-stable": true}' > composer.json ; composer require pantheon-systems/terminus dev-master ; echo 'alias terminus1x=$HOME/terminus1x/vendor/bin/terminus'>>$HOME/.bash_profile ; source $HOME/.bash_profile</code></pre></figure>
+    </div>
+  </div>
+  <!-- Unix: Linux Instructions -->
+  <div role="tabpanel" class="tab-pane" id="linux">
+    <h2 class="tab-content-heading">Install Terminus for Linux</h2>
+    <h3>Requirements</h3>
+    <ul>
+      <li>PHP Version 5.5.9 or later</li>
+      <li><a href="http://www.php-cli.com/">PHP-CLI</a></li>
+      <li><a href="https://getcomposer.org/download">Composer</a></li>
+    </ul>
+    <p class="instruction">Run the following command to install Terminus Alpha (<code>terminus1x</code>) with <a href="https://getcomposer.org/doc/00-intro.md">Composer</a>:</p>
+    <div class="copy-snippet">
+      <button class="btn btn-default btn-clippy" data-clipboard-target="#linux-install-composer"><img class="clippy" src="/source/docs/assets/images/clippy.svg" width="17" alt="Copy to clipboard"></button>
+      <figure><pre id="linux-install-composer"><code class="bash" data-lang="bash">cd $HOME ; mkdir terminus1x ; cd terminus1x ; echo '{"minimum-stability": "dev", "prefer-stable": true}' > composer.json ; composer require pantheon-systems/terminus dev-master ; echo 'alias terminus1x=$HOME/terminus1x/vendor/bin/terminus'>>$HOME/.bashrc ; source $HOME/.bashrc</code></pre></figure>
+    </div>
+  </div>
+  <!-- Windows Instructions -->
+    <div role="tabpanel" class="tab-pane" id="win">
+    <h2 class="tab-content-heading">Install Terminus for Windows</h2>
+    <h3>Requirements</h3>
+    <ul>
+      <li><a href="https://msdn.microsoft.com/en-us/commandline/wsl/install_guide">Bash on Ubuntu on Windows</a></li>
+      <li>PHP Version 5.5.9 or later</li>
+      <li><a href="http://www.php-cli.com/">PHP-CLI</a></li>
+      <li>Composer (<a href="https://getcomposer.org/Composer-Setup.exe">Direct Download</a>)</li>
+    </ul>
+    <p class="instruction">Run the following command to install Terminus Alpha (<code>terminus1x</code>) with <a href="https://getcomposer.org/doc/00-intro.md">Composer</a>:</p>
+    <div class="copy-snippet">
+      <button class="btn btn-default btn-clippy" data-clipboard-target="#win-install-composer"><img class="clippy" src="/source/docs/assets/images/clippy.svg" width="17" alt="Copy to clipboard"></button>
+      <figure><pre id="win-install-composer"><code class="bash" data-lang="bash">cd $HOME ; mkdir terminus1x ; cd terminus1x ; echo '{"minimum-stability": "dev", "prefer-stable": true}' > composer.json ; php composer.phar require pantheon-systems/terminus dev-master ; echo 'alias terminus1x=$HOME/terminus1x/vendor/bin/terminus'>>$HOME/.bash_profile ; source $HOME/.bash_profile</code></pre></figure>
```

Comment:
Here's another set of Composer instructions that should be updated to use `composer init -n` et. al.

Does $HOME/.bash_profile work on Windows?  Maybe Windows 10, but probably not Windows 7.  We should be clear about which versions of Windows we are supporting.